doc/ref_arch/openstack/chapters/chapter05.md

```diff
@@ -22,6 +22,15 @@ the CNTT standard OpenStack version will be Pike.
 The Chapter presents the APIs for the core OpenStack services defined in Chapter 3 and later in the Chapter a
 consolidated view of these and other APIs that would be of interest.
 
+OpenStack is a multi-project framework composed of services evolving independently. It is not enough to rely only on the 
+OpenStack release to characterise the capabilities supported by these services. Regarding OpenStack services APIs, 
+an "API version" is associated to each OpenStack service. 
+In addition to major API versions, some OpenStack services (Nova, Glance, Keystone, Cinder...) support microversions.
+The microversions allow to introduce new features over time. 
+In this chapter, the **major version** and **microversion** are specified per service.
```

Comment:
In OpenStack, when would the major version change?

Reply:
@rabi-abdel  The management of versions depends on services. I didn't find a general OpenStack rule about the change of major version. I suppose it's done for major breakthrough and probably when backward compatibility isn't possible. I'll ask Ildiko.

Reply:
@karinesevilla [Ensuring API Interoperability](https://specs.openstack.org/openstack/api-wg/guidelines/api_interoperability.html) explains when major API versions should be created -- only a guideline. Please see the section, "Evaluating API Changes."

Reply:
happy with the answer

Reply:
@rabi-abdel do you want reference to the API Interoperability added to @karinesevilla description?

Reply:
@pgoyal01 it will be good to have the reference, but I think the can be added later. will approve and merge as is.